modules/local/stimulus_split_csv.nf

```diff
@@ -1,19 +1,25 @@
-
 process STIMULUS_SPLIT_CSV {
-
-    container 'alessiovignoli3/stimulus:latest'
+    
+    tag "$csv"
     label 'process_low'
+    container 'alessiovignoli3/stimulus:latest'
 
     input:
     tuple path(csv), path(parsed_json)
 
     output:
     tuple val("${csv}"), path(parsed_json), path(output), emit: csv_with_split
-    stdout emit: standardout
 
     script:
     output = "${csv.baseName}-${parsed_json.baseName}.csv"
     """
     launch_split_csv.py -c ${csv} -j ${parsed_json} -o ${output}
     """
+
+    stub:
+    output = "${csv.baseName}-${parsed_json.baseName}.csv"
+    """
+    touch ${output}
+    echo "launch_split_csv.py -c ${csv} -j ${parsed_json} -o ${output}" > launch_split_csv.sh
```

Comment:
can you add a comment on why we do this line? for resume?

Reply:
No, this was actually a leak, I removed it